src/main/java/org/ambraproject/wombat/controller/CommentController.java

```diff
@@ -78,11 +78,6 @@
   @Autowired
   private RuntimeConfiguration runtimeConfiguration;
 
-
-  private void addCommentAvailability(Model model) {
-    model.addAttribute("areCommentsDisabled", runtimeConfiguration.areCommentsDisabled());
-  }
```

Comment:
Do you know when this setting might have been used in the past? I wonder if it was a quick way to turn off commenting if somebody was massively spamming us. If that's a concern, do you think removing this as a config value would hinder our ability to respond to those situations? (sorry, there's a lot of hypotheticals there)

Reply:
I was told that the intent was to have an easy switch to disable commenting for whatever reason.  I am unaware that it was ever used but I can see the value.  Might be better to keep the feature and hardcode the config value.

Reply:
If we want to keep it, it should probably be Env var configurable. In the types of scenarios where I imagine we'd want to turn it off (scenarios where some abuse is happening), we'd want to do it in a hurry, and cutting a release to do so would be burdensome.

Reply:
That's a good point.  I'm wary of removing the feature.  Perhaps @shanecyr could weigh in?

Reply:
I'm happy to revert this and make it an optional config with a sensible default (false). I may have gotten overzealous. But I'll wait to hear from Shane.

Reply:
We determined in standup that this feature can burn in hell. Closing out this thread . . .